src/main/scala/coreplex/Configs.scala

```diff
@@ -100,8 +101,8 @@ class BaseCoreplexConfig extends Config (
         TestGeneration.addSuites(rvi.map(_("p")))
         TestGeneration.addSuites((if(site(UseVM)) List("v") else List()).flatMap(env => rvu.map(_(env))))
         TestGeneration.addSuite(benchmarks)
-        List.tabulate(site(NTiles)){ i => (r: Bool, p: Parameters) =>
-          Module(new RocketTile(resetSignal = r)(p.alterPartial({
+        List.tabulate(site(NTiles)){ i => (c: Clock, r: Bool, p: Parameters) =>
+          Module(new RocketTile(clockSignal = c, resetSignal = r)(p.alterPartial({
```

Comment:
Sometimes you use clock/reset, c/r, or clockSignal/resetSignal.  I'm assuming the reason why you've avoided using clock/reset is because those names are used in chisel?  I've used c as an implicit val for configuration case classes (e.g. CoreplexConfig), so I'd avoid that as well.  clockSignal and resetSignal are long, so that's not good.  How about using _clock and _reset?  This is something that Chisel3 used as function arguments.  Whatever you pick, you should be consistent throughout the codebase.